src/test/java/core/basesyntax/InputValidatorImplTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.InputValidator;
+import core.basesyntax.service.InputValidatorImpl;
+import java.util.List;
+import org.junit.Test;
+
+public class InputValidatorImplTest {
+    private static final String HEADER = "type,fruit,quantity";
+    private static final List<String> listOk = List.of(HEADER, "s,banana,100");
+    private static final List<String> listNotOk1 = List.of(HEADER, ",banana,100");
```

Comment:
we can improve naming here :
```suggestion
    private static final List<String> invaliOperationList = List.of(HEADER, ",banana,100");
```
same for other input names